vayesta/core/ao2mo/helper.py

```diff
@@ -130,12 +240,79 @@ def pack_vvvv(vvvv):
     nvir = vvvv.shape[0]
     return pyscf.ao2mo.restore(4, vvvv, nvir)
 
-def get_block(eris, block):
-    if block == 'ovvv':
-        return get_ovvv(eris)
-    if block == 'vvvv':
-        return get_vvvv(eris)
-    return getattr(eris, block)
+def contract_dm2_eris(dm2, eris):
+    """ Contracts _ChemistsERIs with the two body density matrix
+
+    Parameters
+    ----------
+    dm2  : ndarry or (ndarray, ndarray, ndarry)
+        Two body density matrix or tuple of aa, ab, bb blocks for UHF
+    eris : _ChemistERIs
+        PySCF ERIs object
+
+    Returns
+    ----------
+    e2   : float
+        Two body energy contribution
+    """
```

Comment:
I think it's this docstrings which needs to be corrected, i.e. spaces removed

Reply:
Okay, I've fixed that too.